src/App.css

```diff
@@ -29,20 +31,41 @@
   }
   
   /* Button alignment */
-  .login-button {
+  .button-custom {
     display: flex;
     justify-content: center;
     align-items: center;
     margin-top: 1rem;
     width: 100%;
+    padding: 10px; 
+    border-radius: 8px; 
+    font-weight: bold; 
+    border: none; 
```

Comment:
you are using tailwind CSS, so why are there too many custom styles? @Himanshusingh770